src/main/java/Lubot.java

```diff
@@ -0,0 +1,202 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Lubot {
+	private static ArrayList<Task> tasks = new ArrayList<>();
+	private static int n = 0;
+
+    public static void main(String[] args) {
+        String logo = ".____         ___.           __   \n"
+        + "|    |    __ _\\_ |__   _____/  |_ \n"
+        + "|    |   |  |  \\ __ \\ /  _ \\   __\\\n"
+        + "|    |___|  |  / \\_\\ (  <_> )  |  \n"
+        + "|_______ \\____/|___  /\\____/|__|  \n"
+        + "        \\/         \\/             \n";
```

Comment:
Should line wrap have 8 spaces indentation?

```suggestion
        String logo = ".____         ___.           __   \n"
                + "|    |    __ _\\_ |__   _____/  |_ \n"
                + "|    |   |  |  \\ __ \\ /  _ \\   __\\\n"
                + "|    |___|  |  / \\_\\ (  <_> )  |  \n"
                + "|_______ \\____/|___  /\\____/|__|  \n"
                + "        \\/         \\/             \n";
```

I like the ascii art!